test/jdk/java/time/test/java/time/TestZoneOffset.java

```diff
@@ -82,4 +83,18 @@ public void test_factory_ofTotalSecondsSame() {
         assertSame(ZoneOffset.ofTotalSeconds(0), ZoneOffset.UTC);
     }
 
+    @Test
+    public void test_quarter_cache() throws Exception {
+        for (int hour = -18; hour < 18; hour++) {
```

Comment:
The offset range is inclusive, so you may want to test +18 hour as well, since it is the edge case.